### DISCUSSION TITLE
Add admin control for toggling Did "read only mode"

### DISCUSSION THREAD

**Discussion by okms** [Puzzlepart/did#895] [Feature request]
- [x] I have searched for existing issues to avoid creating duplicates

### TLDR
We need to be able to temporarily disable confirmation (and un-confirmation) of weeks. This is useful when doing upgrades of the underlying table store, and other backend work

Ideally we should disable all types of Create, Update and Delete operations also creating projects, customers etc. 

Simplest solution: disable only week confirmation and unconfirmation
More advanced 
- disable week confirmation and unconfirmation
- project creation and updates
- customer creation and updates
- label creation, update

**Comment by olemp** [comment]
Are we thinking on a per-subscription basis? 

cc @damsleth @okms

**Comment by damsleth** [comment]
> Are we thinking on a per-subscription basis?
> 
> cc @damsleth @okms

We should at least have one mode for "all subscriptions", for use during upgrades etc.

**Comment by olemp** [comment]
> > Are we thinking on a per-subscription basis?
> > cc @damsleth @okms
> 
> We should at least have one mode for "all subscriptions", for use during upgrades etc.

That would be the easiest. Then it could just be an application setting.
